docs/online-experiments/firebase.md

```diff
@@ -0,0 +1,111 @@
+# Firebase Integration
+
+Here we explain how to set up a [Firebase](https://firebase.google.com/) website to collect observations for an AutoRA workflow. [Here](user-guide/experiment-runners/experimentation-managers/firebase/) you find information on how to connect AutoRA to a Firebase website. Additionally, [here](user-guide/experiment-runners/recruitment-managers/prolific/) we explain how to include [Prolific](https://www.prolific.co/) to automatically recruit participants.
+
+For setting up the online experiment, we recommend using either the [user cookiecutter template](https://github.com/AutoResearch/autora-user-cookiecutter) or the [create-react-app template](https://github.com/AutoResearch/cra-template-autora-firebase).
+
+!!! hint
+    The cookiecutter template also provides a template for the AutoRA workflow used in online experiments.
+
+## Installation
+
+To make sure that node is installed on your system, run the following command. 
+```shell
+node -v
+```
+If you don't see a version number or the version number is bellow 16.0, install or update node following the instruction on the [node.js website](https://nodejs.org/en/).
+
+When node is available on your system, you can use ***create-react-app*** by running the following command.
+```shell
+npx create-react-app path/to/react/pp --template autora-firebase
+```
+If you want to use ***cookiecutter***, run the following command and follow the instructions.
+```shell
+cookiecutter https://github.com/AutoResearch/autora-user-cookiecutter
+```
+This creates your ***project folder***. Before writing code for your website, you also need to set up a Firebase project.
+ 
+## Setting up a Firebase project
```

Comment:
```suggestion
## Firebase Project Setup
```

Reply:
Are we set on capitalizing all headers (no matter what level?). If so, we need to make some changes to existing documentation. For example, [https://autoresearch.github.io/autora/user-guide/theorists/darts/meta-parameters/](https://autoresearch.github.io/autora/user-guide/theorists/darts/meta-parameters/)

Reply:
personally, i really don't like the ambiguity introduced when capitalizing only "major" words. i would much rather do sentence case (only first letter of first word, no matter what) or strong title case (every first letter captialized)

Reply:
Yes, you are right about that. I don't like it, either. The question is, do we want to make a distinction between the first header (title of the page) and the other headers? (Maybe strong title case in the title of the page and sentence case in all other headers)
Or should we have the same casing in all titles no matter if it is a title of the page or a header inside the page? (In that case I would personaly prefer sentence case)

Reply:
Oh sorry, you answered by saying "globaly". Makes sense to me!